### PR TITLE
{Compute} `az vm/vmss create`: Update notification message for Trusted Launch by default

### PR DESCRIPTION
**Pull request by zhoxing-ms**
**Related command**

`az vm/vmss create`
`az disk create`

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Compute

**Review by yanzhudd — APPROVED** [review]

**Review by yanzhudd — DISMISSED** [review]

File: src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py [hunk 1]
```diff
@@ -554,9 +554,9 @@ def trusted_launch_warning_log(namespace, generation_version, features):
     if not generation_version:
         return
 
-    log_message = 'Starting Build 2023 event, "az vm/vmss create" command will deploy Trusted Launch VM by default.' \
-                  ' To know more about Trusted Launch, please visit' \
-                  ' https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch'
+    log_message = 'Ignite 2023 onwards "az vm/vmss create" command will deploy Gen2-Trusted ' \
+                  'Launch VM by default. To know more about the default change and Trusted Launch, ' \
+                  'please visit https://aka.ms/TLaD'
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py:559]:
```suggestion
    log_message = 'Ignite (November) 2023 onwards "az vm/vmss create" command will deploy Gen2-Trusted ' \
                  'Launch VM by default. To know more about the default change and Trusted Launch, ' \
                  'please visit https://aka.ms/TLaD'
```

File: src/azure-cli/azure/cli/command_modules/vm/custom.py [hunk 2]
```diff
@@ -340,9 +340,9 @@ def create_managed_disk(cmd, resource_group_name, disk_name, location=None,  # p
         raise RequiredArgumentMissingError(
             'usage error: --upload-size-bytes should be used together with --upload-type')
 
-    log_message = 'Starting Build 2023 event, "az disk create" command will deploy Trusted Launch VM by default.' \
-                  ' To know more about Trusted Launch, please visit' \
-                  ' https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch'
+    log_message = 'Ignite 2023 onwards "az disk create" command will deploy Gen2-Trusted Launch VM ' \
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/custom.py:343]:
```suggestion
    log_message = 'Ignite (November) 2023 onwards "az disk create" command will deploy Gen2-Trusted Launch VM ' \
```

File: src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py [hunk 3]
```diff
@@ -554,9 +554,9 @@ def trusted_launch_warning_log(namespace, generation_version, features):
     if not generation_version:
         return
 
-    log_message = 'Starting Build 2023 event, "az vm/vmss create" command will deploy Trusted Launch VM by default.' \
-                  ' To know more about Trusted Launch, please visit' \
-                  ' https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch'
+    log_message = 'Ignite 2023 onwards "az vm/vmss create" command will deploy Gen2-Trusted ' \
```
[the remark this answers is not in the document]
Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py]: Update

[the remark this answers is not in the document]
Comment (AjKundnani) [reply src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py]: Looks good. Thanks. Approved my review.

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**